static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -131,6 +138,11 @@ define([
                 players[atomId].endSlate = getEndSlate(overlay);
             }
         }
+
+        if (closest(players[atomId].iframe, '.immersive-main-media__media')) {
```

Comment:
You could just pass iframe as the first argument to closest here rather than players[atomId].iframe, it's passed as an argument to onPlayerReady so you don't need to get it from players[atomId].